packages/peregrine/lib/talons/CheckoutPage/PaymentInformation/usePaymentInformation.js

```diff
@@ -73,7 +75,8 @@ export const usePaymentInformation = props => {
         data: paymentInformationData,
         loading: paymentInformationLoading
     } = useQuery(getPaymentInformation, {
-        variables: { cartId }
+        variables: { cartId },
+        fetchPolicy: 'cache-and-network'
```

Comment:
This has introduced another "bug" in that there we get a render cycle where we aren't loading since it uses the cache, which causes the CC/braintree components to render causing a few issues like the nonce is deleted and there is an error in console because the CC component renders, then unmounts, then renders.